modules/bot/src/agents/bot.ts

```diff
@@ -53,8 +53,8 @@ export default {
     const nodeUrl = process.env.INDRA_NODE_URL;
     const messagingUrl = process.env.INDRA_NATS_URL;
 
-    const randomInterval = Math.round(argv.interval * 0.75 + Math.random() * (argv.interval * 0.5));
-    log.info(`Using random inteval: ${randomInterval}`);
+    const randomInterval = argv.interval;
```

Comment:
This code is presented here to show how I generated the benchmark numbers. I will revert it prior to merge.